optimum/intel/openvino/modeling_decoder.py

```diff
@@ -543,7 +574,7 @@ def _from_pretrained(
             local_files_only=local_files_only,
         )
 
-        model = cls.load_model(model_cache_path, load_in_8bit=load_in_8bit)
+        model = cls.load_model(model_cache_path, load_in_8bit=False if load_in_4bit else load_in_8bit)
```

Comment:
should we raise an error to ensure `load_in_4bit` and `load_in_8bit` are not both set to True

Reply:
sure